app/Models/AnonymousQuestions/AnswerSheet.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Models\AnonymousQuestions;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+use Illuminate\Database\Eloquent\Model;
+
+use App\Models\User;
+use App\Models\Semester;
+use App\Models\GeneralAssemblies\QuestionOption;
+use App\Models\AnonymousQuestions\LongAnswer;
+
+/**
+ * Groups a collegist's answers to anonymous questions
```

Comment:
Okay I think I see now. Is this only to group the different questions filled by the same user? Then would not that be simpler to group them based on a random number? Anything that can not be linked to the user would do. Just store the same value for the same user, and since the data can not change, you can group them anytime later.

Reply:
It also contains some anonymous information about the user. For now, it only contains the year of acceptance; but if we revise the role system, we will be able to add whether they reside in the building.

Reply:
My argument is that if we want we can ask about these information if they are relevant. But I see the point.

Reply:
but we loose authenticity that it is anonymous